truss-chains/truss_chains/definitions.py

```diff
@@ -75,7 +75,6 @@ class SafeModel(pydantic.BaseModel):
         arbitrary_types_allowed=False,
         strict=True,
         validate_assignment=True,
-        extra="forbid",
```

Comment:
Removing due to: https://github.com/basetenlabs/truss/blob/ea9b45ab610a509477bfc8637641452e9c537506/truss-chains/truss_chains/model_skeleton.py#L23-L27

Per discussion with @marius-baseten, it makes sense to remove this at `SafeModel` level since there can be future discrepancies between Truss CLI code gen and Context Builder.